package.json

```diff
@@ -26,7 +26,7 @@
     "fixjs": "eslint src --fix",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint src",
-    "test": "jest --forceExit --detectOpenHandles",
+    "test": "jest --forceExit --detectOpenHandles --no-cache",
```

Comment:
Can we create a new command just for the CI? This will significantly increase the duration of the test during local development.

Reply:
Good point, created new issue for it - https://github.com/RedHatInsights/insights-chrome/issues/1745